hydra/functions/FlatteLineShape.h

```diff
@@ -115,6 +120,18 @@ namespace hydra {
             return *this;
         }
 
+        __hydra_host__  __hydra_device__ inline
+        std::vector<std::vector<double>>GetParams() const {
+            return fParams;
+        }
+
+        __hydra_host__  __hydra_device__ inline
+        void SetDaughter1Mass(std::vector<std::vector<double>> _params) {
```

Comment:
it seems to me this member is setting the wrong class member... Should be `fDaughterMass1` instead of `fParams`, right?